Updated the Weheat data rate documentation (#37715)
home-assistant/home-assistant.github.io: source/_integrations/weheat.markdown

## Data updates

> The integration uses {% term polling %} to retrieve data logs every 120 seconds for a single heat pump. This interval increases proportionally with the number of heat pumps, for example, to 240 seconds for two heat pumps. Additionally, energy logs are retrieved from the cloud every 1800 seconds.

Reviewer: What are data logs and energy logs? When i hear logs I'm thinking about a .log or .txt file.

Reviewer 2: @joostlek agreed, how about:

```suggestion
The integration uses {% term polling %} to retrieve data every 120 seconds for a single heat pump. This interval increases proportionally with the number of heat pumps, for example, to 240 seconds for two heat pumps. Additionally, energy data is retrieved from the cloud every 1800 seconds.
```

Reviewer: Looks good! But i dont have permission to commit it

Author: I understand the confusion, committed the suggestion.
